front50-web/src/main/groovy/com/netflix/spinnaker/front50/controllers/v2/EntityTagsController.groovy

```diff
@@ -58,6 +58,18 @@ class EntityTagsController {
     return taggedEntityDAO?.all()?.findAll { prefix ? it.id.startsWith(prefix) : true }
   }
 
+  @RequestMapping(value = "/bulkFetch", method = RequestMethod.POST)
+  Set<EntityTags> bulkFetch(@RequestBody final Collection<String> ids) {
+    return findAllByIds(ids)
+  }
+
+  @RequestMapping(value = "/bulkDelete", method = RequestMethod.POST)
+  void bulkDelete(@RequestBody final Collection<String> ids) {
+    ids.each {
+      taggedEntityDAO.delete(it)
```

Comment:
What happens if one of the individual deletes fails? Any perceived fallout?

I think in the current delete flow that we could have an entitytag removed from `elasticsearch` and a subsequent failure in `front50`. 

Not sure that's been happening but something to keep in mind especially if we start ramping up the number of tags being modified/deleted in a single operation.